src/lib/DAL/CardMethods.js

```diff
@@ -19,6 +19,15 @@ CardMethods.DeleteCard = function(id) {
     Cards.remove(id);
 };
 
+CardMethods.SubmitComment = function(id,comment) {
+    var card = Cards.findOne({_id:id});
+    var oldComments = card.comments;
+
+    oldComments.push(comment);
+    oldComments.reverse();
```

Comment:
what is this?  I thinks comments should be in order of comment made personally, it is the flow of most other commenting systems across the web outside of youtube.  Wouldn't this happen the as new comments are being created (comment = numbers) as new comments come in:
[] -> [1] -> [2, 1] -> [3, 1, 2] -> [4, 2, 1, 3] -> [5, 3, 1, 2, 4]
what I believe you are trying to do (incoming comment is 1st comment) should be:
oldComents.reverse();
oldComments.push(comment);
oldComments.reverse();
I think it should always just push the new comment without any reverses, but its whatever the PO wants.